Access Medicine.js

```diff
@@ -0,0 +1,343 @@
+			var bookElement = ZU.xpath(section, './/p')[0];
+			var bookTitle = ZU.trimInternal(bookElement.textContent);
+			var sectionId = ZU.xpath(titleElement,'.//a')[0].href;
+			//sectionId is first query element in url
+			var beginCut = sectionId.indexOf("=");
+			var endCut = sectionId.indexOf("&");
+			title = title+" ("+bookTitle+")";
+			sectionId = sectionId.substring(beginCut+1, endCut);
+			var link = baseCitation+sectionId;
+			//prevent overriding, keep most relevant title
+			if(!sectionDict[link]){
+				sectionDict[link]=title;
+			}
+		}
+		Z.selectItems(sectionDict, function(selected){
+			if(!selected){
+				return;
+			}
+			for (var link in selected){
+				//got weird results with ZU.processDocuments
+				//using a little hack instead
+				risTranslate(doc, link, false);
+			}
+		});
+	}
+	else{
+		//only book section from now on
+		var pattern = /\/content.*/i;
+		var baseUrl = url.replace(pattern,'');
+		var baseCitation = baseUrl + "/downloadCitation.aspx?format=ris&sectionid=";
+		var sectionId = url.split("sectionid=")[1];
+		var link = baseCitation + sectionId;
+		risTranslate(doc, link, true);
+	}
+}
+
+function risTranslate(doc, link, bookSection){
+	Zotero.Utilities.doGet(link, function(risText){
+		//set RIS import translator
+		var translator = Zotero.loadTranslator("import");
+		translator.setTranslator("32d59d2d-b65a-4da4-b0a3-bdd3cfb979e7");
+		translator.setString(risText);
+		translator.setHandler("itemDone", function(obj, item) {
+			if(bookSection){
+				var chapterSpan = doc.getElementById('pageContent_lblChapterTitle1');
+				if (chapterSpan) {
+					//remove 'chapter' text if present
+					var chapterText = chapterSpan.innerHTML.trim();
+					//item.notes.push('Chapter: '+chapterText);
+					item.notes.push({note:chapterText});
+				}
+				item.attachments.push({
+					title: "Snapshot",
+					document: doc
+				});
+			}
+			//parse out edition from title
+			var bookTitle = item.bookTitle;
+			if(bookTitle.indexOf(",") != -1){
+				//get last substring (book title might have commas in it)
+				var splitOnComma = bookTitle.split(",");
+				var len = splitOnComma.length;
+				var edition = splitOnComma[len-1];
+				//remove e
+				if(edition.indexOf("e") != -1){
+					edition = edition.replace("e","");
+				}
+				edition = edition.trim();
+				item.edition=edition;
+				//rebuild book title
+				splitOnComma.splice(-1);
+				var newBookTitle=splitOnComma.join(",");
+				item.bookTitle=newBookTitle;
+			}
+
+			
+			if (item.abstractNote) {
+				//make sure abstract text has spaces after periods
+				var	adjustedAbstract = item.abstractNote.replace('.','. ');
+				//remove extra spaces from abstract notes field (common throughout)
+  				adjustedAbstract = adjustedAbstract.replace(/\s+/g, ' ');
+  				item.abstractNote = adjustedAbstract;
+			}
+
+			//remove numberOfVolumes, incorrectly set in RIS
+			item.numberOfVolumes="";
+
+			//remove authors if they dont have a first and last name
```

Comment:
My only remaining concern is here: this would remove any institutional authors (e.g. "World Health Organization"). Are we pretty sure the database doesn't have any of those?